src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        switch (new Random().nextInt(4)) {
```

Comment:
4 is a magic number https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names